dashboard/app/assets/stylesheets/application.scss

```diff
@@ -538,6 +539,22 @@ $default-modal-width: 640px;
       float: left;
       position: relative;
     }
+
+    .hint {
+      padding: 15px 15px 15px 40px;
+      background-color: $hint-bgcolor;
+      border-radius: 10px;
+    }
+    .hintInner {
```

Comment:
I actually much prefer the new version, and not just because it fixes that annoying border-runs-through-dog-image bug.

Might be worth quickly running this by whoever owns these levels, though; Kiki, I would guess.